transcom/mymove
# B-18162 add violations to pws_violations table

[B-18162](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871843)
Summary
Add new entries to pws_violations table for QAE reports, per conversation with PO. Spreadsheet with changes is attached to backlog item.

How to test
1. Login in a QAE role
2. Access any move
3. Click the Quality Assurance tab
4. Click Create Report in the Counseling QAE Reports section
5. Fill out the required info and advance
6. Confirm that the new entries exist in the "Inventory & Documentation" and "Packing/Unpacking" sub categories of Physical Move Services, and are located at the end of their respective sections
7. When expanding the new entries, confirm that the detailed text matches the spreadsheet submitted by the PO
8. Select some or all of the new entries and confirm you can save and submit the report
9. From the Quality Assurance tab screen, now click on Create Report in the Shipment QAE Reports section
10. Fill out the required info and advance
11. Confirm that the new entries exist in the "Inventory & Documentation" and "Packing/Unpacking" sub categories of Physical Move Services, and are located at the end of their respective sections
12. When expanding the new entries, confirm that the detailed text matches the spreadsheet submitted by the PO
13. Select some or all of the new entries and confirm you can save and submit the report
